templates/assets/cloudformation/service-ecs.yml

```diff
@@ -232,7 +239,11 @@ Conditions:
       - "Fn::Equals":
         - !Ref TaskCpu
         - ''
-
+  HasPrivilegedMode:
```

Comment:
can you remove the condition and just pass the value directly in to the `Privileged` attribute?